EOLib/Domain/Map/CurrentMapStateRepository.cs

```diff
@@ -46,6 +46,8 @@ public interface ICurrentMapStateProvider
 
         short JailMapID { get; }
 
+        bool IsJail { get; }
```

Comment:
Even if it's a readonly propery, add this to `ICurrentMapStateRepository` too for consistency

Reply:
done